### PR TITLE
🐛Update amp-playbuzz internal iframe url

### PR DESCRIPTION
**Pull request by iddanh**
PR for https://github.com/ampproject/amphtml/issues/30525
- Change iframe url from www.playbuzz.com to app.ex.co

### CODE REVIEW

**Comment by iddanh** [comment]
@krdwan Did you have a chance to look at this PR?

**Comment by rcebulko** [comment]
@krdwan bump!

**Review by krdwan — APPROVED** [review]
LGTM, congrats on the rebrand

**Comment by krdwan** [comment]
@iddanh Thanks for the PR, just taking a look at some of the automated checks that are failing.  Could you try rebase to a fresh master and push?  I think the particular version of master you were might be causing the failures.  For me the set of commands is something like this:

git checkout master
git pull
git checkout [this branchname]
git rebase -i master
git push origin [this branchname] --force

**Comment by iddanh** [comment]
@krdwan Thanks! 😁 

I merged master and fixed some failing tests and lint issues. 
I think the remaining failures are not connected to my changes, right?

**Comment by krdwan** [comment]
Thanks @iddanh the remaining failures don't appear to be related to your changes.  Let me look into them and follow up if there's anything else needed from your end.

File: extensions/amp-playbuzz/0.1/test/test-amp-playbuzz.js [hunk 1]
```diff
@@ -104,30 +100,30 @@ describes.realWin(
 
     function testIframe(iframe, itemSrcUrl) {
       expect(iframe).to.not.be.null;
-      expect(startsWith(iframe.src, itemSrcUrl)).to.be.true;
+      expect(iframe.src.includes(itemSrcUrl)).to.be.true;
```
Comment (rcebulko) [inline extensions/amp-playbuzz/0.1/test/test-amp-playbuzz.js:103]:
```suggestion
      expect(iframe.src.startsWith(itemSrcUrl)).to.be.true;
```

Comment (iddanh) [reply extensions/amp-playbuzz/0.1/test/test-amp-playbuzz.js:103]: Following the changes I made to the extension, the item src url can be formatted as either:

**Pull request merged**